lib/rest/lib/params.c

```diff
@@ -3,21 +3,20 @@
  * Copyright (C) 2022 Micron Technology, Inc.  All rights reserved.
  */
 
+#include <event.h>
```

Comment:
whoops

Reply:
i don't understand the issue.  looks like this now:
```
#include <event.h>
#include <stddef.h>
#include <stdlib.h>
#include <string.h>

#include <event2/http.h>
#include <sys/queue.h>

#include <hse/rest/params.h>
#include <hse/util/assert.h>
#include <hse/util/compiler.h>
```